common/middleware/set-services.js

```diff
@@ -0,0 +1,26 @@
+const services = require('../services')
+
+const setServices = (req, res, next) => {
+  const servicesRequestContext = {}
+
+  req.services = Object.keys(services).reduce((acc, serviceKey) => {
```

Comment:
Now we're dynamically adding services, I wonder if adding a small debug logger line to indicate what's been added, might be useful to understand what's happening?